### PR TITLE
[TW-1696] Deprecate Live Collections SDK docs

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Removes topics and reference related to Live Collections with SDK, adds redirects to the Collections overview

### CODE REVIEW

File: src/pages/docs/publishing-your-api/authoring-your-documentation.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ You can embed videos hosted on YouTube or Vimeo in your documentation. Your docu
 * To embed a video using the Markdown editor, use the following syntax:
 
     ```md
-    <video src="https://youtube.com/embed/1xTTNGacuaQ" alt="Set Up Live Collections" width="340" height="170"></video>
+    <video src="https://www.youtube.com/watch?v=FeqSWgx6FxY" alt="View Public Workspace Metrics" width="340" height="170"></video>
```
Comment (nerminamiller-postman) [inline src/pages/docs/publishing-your-api/authoring-your-documentation.md:211]:
```suggestion
    <video src="https://www.youtube.com/embed/FeqSWgx6FxY?si=-5pR5tgUQtPN8P6z" alt="View Public Workspace Metrics" width="340" height="170"></video>
```

**Review by mark-anthony-postman — APPROVED** [review]
These changes LGTM!

However, I also see a reference to Live Collections here that I think you need to remove: https://learning.postman.com/docs/getting-started/basics/navigating-postman/#right-sidebar

**Comment by nerminamiller-postman** [comment]
@mark-anthony-postman thank you for your helpful review! Mind checking that embed link in the API publishing topic to make sure I formatted it properly?

**Comment by mark-anthony-postman** [comment]
> @mark-anthony-postman thank you for your helpful review! Mind checking that embed link in the API publishing topic to make sure I formatted it properly?

It looked good to me--I tried it out in some sample documentation and it worked just fine

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**